.pipelines/templates/ToolchainBuild.yml

```diff
@@ -27,7 +39,24 @@ steps:
     parameters:
       buildRepoRoot: ${{ parameters.buildRepoRoot }}
 
-  - bash: sudo make -C "${{ parameters.buildRepoRoot }}/toolkit" "-j$(nproc)" toolchain QUICK_REBUILD=y
+  - bash: |
+      if [[ ${{ parameters.dailyBuildID }} == "lkg" ]]; then
+        daily_build_id_arg="DAILY_BUILD_ID=${{ parameters.dailyBuildID }}"
+      elif [[ -n "${{ parameters.dailyBuildID }}" ]]; then
+        daily_build_id_arg=""
+      fi
```

Comment:
Why do we want to set the arg to an empty string if `dailyBuildID` is not an empty string but it's different than "lkg"? Also no need to set an empty string - uninitialized variables are empty by default in Bash. Unless you use `set -u`, in which case using them is considered an error.:)

Reply:
Also, I think instead of `if [[ ${{ parameters.dailyBuildID }} == "lkg" ]];` we're more interested in `if [[ -n "${{ parameters.dailyBuildID }}" ]];`.

Reply:
Updated